src/emailAlert.ts

```diff
@@ -15,12 +22,18 @@ const ALERT_TO_EMAIL = process.env.ALERT_TO_EMAIL
 
 sendgrid.setApiKey(SENDGRID_API_KEY || '')
 
-export const sendEmailAlert = (message: string) => {
+// limit email alerts to 1 per 30min
+export const sendEmailAlert = async (message: string) => {
     if (!(SENDGRID_API_KEY && ALERT_FROM_EMAIL && ALERT_TO_EMAIL)) {
-        log('Email Alerts not configured, not sending email alert.')
+        log('Email alert not sent - not configured')
         return
     }
 
+    const wasBlocked = await rollingLimiter.limit('register')
+    if (wasBlocked) {
+        return log('Email alert not sent - throttling');
+    }
```

Comment:
Also: You used the same key that was used for member registration throttling - `rollingLimiter.limit('register')`

Reply:
> Also: You used the same key that was used for member registration throttling - `rollingLimiter.limit('register')`

copy pasta :) I think since it is a separate instance of the limiter I think it shouldn't have caused an issue, but I will certainly update it for code clarity.